<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tech_filter</class>
 <widget class="QDialog" name="tech_filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(132, 169, 172);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>391</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);
</string>
   </property>
   <property name="title">
    <string>تکنولوژی</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(32, 64, 81);
color: rgb(228, 227, 227);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);
</string>
        </property>
        <property name="text">
         <string>نام محصول:     </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);
</string>
        </property>
        <property name="text">
         <string>تومان</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(32, 64, 81);
color: rgb(228, 227, 227);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);
</string>
        </property>
        <property name="text">
         <string>تا</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(32, 64, 81);
color: rgb(228, 227, 227);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);
</string>
        </property>
        <property name="text">
         <string>قیمت :                  </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(32, 64, 81);
color: rgb(228, 227, 227);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);
</string>
        </property>
        <property name="text">
         <string>نام فروشنده:   </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(32, 64, 81);
color: rgb(228, 227, 227);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <item>
         <property name="text">
          <string>انتخاب کنید</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>موبایل</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>لپ تاپ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>کامپیوتر</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>تجهیزات جانبی</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>دوربین</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ساعت هوشمند</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>کتابخوان</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>تجهیزات شبکه</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ماشین های اداری</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);
</string>
        </property>
        <property name="text">
         <string>دسته بندی: </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(32, 64, 81);
color: rgb(228, 227, 227);
font: 11pt &quot;Lalezar&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lalezar&quot;;
color: rgb(228, 227, 227);
</string>
        </property>
        <property name="text">
         <string>کلمات کلیدی:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>87</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 64, 81);
color: rgb(228, 227, 227);
font: 11pt &quot;Lalezar&quot;;</string>
   </property>
   <property name="text">
    <string>فیلتر کردن</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>89</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32, 64, 81);
color: rgb(228, 227, 227);
font: 11pt &quot;Lalezar&quot;;</string>
   </property>
   <property name="text">
    <string>حذف فیلتر</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
